spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConnectorUserAgentProvider.java

```diff
@@ -100,4 +101,13 @@ static Optional<String> getGcpRegion() {
   public String getUserAgent() {
     return USER_AGENT + " datasource/" + dataSourceVersion;
   }
+
+  @Override
+  public String getConnectorInfo() {
+    JsonObject jsonObject = new JsonObject();
+    jsonObject.addProperty("connectorVersion", SparkBigQueryUtil.CONNECTOR_VERSION);
```

Comment:
Why are the other data points not added here?

Reply:
What other data points do we need here?